experimenter/tests/integration/nimbus/test_remote_settings.py

```diff
@@ -254,3 +255,32 @@ def test_rollout_live_update_reject_on_experimenter(
     summary_page.set_rejection_reason()
     summary_page.submit_rejection()
     summary_page.wait_for_rejection_notice_visible()
+
+
+@pytest.mark.remote_settings
+@pytest.mark.skipif(
+    any(
+        app in os.getenv("PYTEST_ARGS")
+        for app in ["FOCUS_IOS", "FIREFOX_IOS", "FENIX", "FOCUS_ANDROID"]
+    ),
```

Comment:
I mean, since we only want to skip desktop all you need to do is 

```py
os.getenv("PYTEST_ARGS") != 'DESKTOP'
```

Reply:
oh duh, elise. 🤦‍♀️

Reply:
But just for the sake of argument, say you wanted to skip two, then yes what you're doing is a way to do it but a more succinct way to do it would be with the best data type of all time, `set`

```py
{"DESKTOP", "FENIX"}.intersection({"FOCUS_IOS", "FIREFOX_IOS", "FENIX", "FOCUS_ANDROID"})
```

Reply:
Jared loves his ```set```'s

Reply:
🤣